protocol/0044-LIME-lp_mechanics.md

```diff
@@ -335,11 +340,34 @@ In the case of spot markets it will be transferred into the network treasury for
 - For a market that is in continuous trading and a single LP has committed liquidity:
   - The LP can cancel their commitment at any time (though this may involve incurring a penalty) (<a name="0044-LIME-060" href="#0044-LIME-060">0044-LIME-060</a>) for spot (<a name="0044-LIME-056" href="#0044-LIME-056">0044-LIME-056</a>)
+- For a market that is in continuous trading and LP has committed liquidity
+  - when `market.liquidity.providers.fee.calculationTimeStep` is set to `0` any funds that are in `ACCOUNT_TYPE_FEES_LIQUIDITY` account will be distributed to `ACCOUNT_TYPE_LP_LIQUIDITY_FEES` on the next block (<a name="0044-LIME-061" href="#0044-LIME-061">0044-LIME-061</a>)
+  - if `market.liquidity.providers.fee.calculationTimeStep` is set to `10s` and `validators.epoch.length` is set to `15s`, during the first `10` seconds of the current epoch parameter change `market.liquidity.providers.fee.calculationTimeStep = 3s` is enacted, at the end of the epoch any funds that are in `ACCOUNT_TYPE_FEES_LIQUIDITY` account will be distributed to `ACCOUNT_TYPE_LP_LIQUIDITY_FEES` on the next block. For the next epoch the distribution will take place at `3` second intervals (<a name="0044-LIME-062" href="#0044-LIME-062">0044-LIME-062</a>)
+- For a market that is in continuous trading if a new LP has active buy and sell orders on the market then makes a liquidity commitment to that market, at the start of the next epoch the active orders will count towards the LPs liquidity commitment. (<a name="0044-LIME-071" href="#0044-LIME-071">0044-LIME-071</a>)
+- If an LP with a liquidity provision and active orders on a market cancel their provision only (orders remain active), after the cancellation penalty (if it applies at end of epoch) the end of next epoch lp will not accrue any rewards or incur penalty for trades on the market. (<a name="0044-LIME-081" href="#0044-LIME-081">0044-LIME-081</a>)
+- For a market with market parameter `market.liquidity.priceRange = 0.05` (5%), is in continuous trading with `mid price` set to `5`, a new LP has committed liquidity with buy order at price `4.74` and sell order at price `5.25`. During the epoch market parameter change `market.liquidity.priceRange = 0.1` (10%) is enacted then at the end of the current epoch the LP is meeting their committed volume of notional and a bond penalty will apply. (<a name="0044-LIME-072" href="#0044-LIME-072">0044-LIME-072</a>)
+
+- For a market with parameter `market.liquidity.priceRange = 0.05` (5%), is in continuous trading with `mid price` set to `5`, a new LP has committed liquidity and orders at buy price `4.74` and sell price `5.25`. During the epoch market parameter change `market.liquidity.priceRange = 0.01` (1%) is enacted then at the end of the current epoch the LP is meeting their volume of notional and penalty will not apply. (<a name="0044-LIME-074" href="#0044-LIME-074">0044-LIME-074</a>)
```

Comment:
This one, they're not meeting the commitment during the entire epoch, the param change can't help, they get slashed.

Reply:
I've stopped reading the rest, please review the remaining in the light of the above comments.

Reply:
I made an update on the Acs